app/models/user.rb

```diff
@@ -0,0 +1,20 @@
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable,
```

Comment:
:recoverable implies sending emails right? We might want to either disable this for now, or set up the email config otherwise.

Reply:
I would disable it for now, but create a separate ticket for :recoverable and m.b. :confirmable - I would like to send users welcome-email (but without confirmation link)